framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/OpenShiftDeployment.java

```diff
@@ -92,6 +94,18 @@ public String getDeploymentConfigName() {
         return getServiceName();
     }
 
+    protected String getDeploymentConfigName(OpenShift openShift, Pattern regexp) {
+        // Try to find deployment config name from all available deployment configs
+        List<DeploymentConfig> deploymentConfigs = openShift.getDeploymentConfigs();
+        for (DeploymentConfig deploymentConfig : deploymentConfigs) {
+            if (regexp.matcher(deploymentConfig.getMetadata().getName()).matches()) {
```

Comment:
What about case when is more then one suitable match found? I think it would be better to replace this with stream witch filter out matches and if there are more then one we should throw some exception.

Reply:
Implemented